centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl

```diff
@@ -25,8 +29,9 @@
     </contributor>
     <category citation-format="note"/>
     <category field="social_science"/>
-    <updated>2016-02-19T12:00:00+00:00</updated>
-    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <updated>2016-03-14T12:00:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a
+      Creative Commons Attribution-ShareAlike 3.0 License</rights>
```

Comment:
@PhilPons, you can't have a line break here (and we rather not have them anywhere else either, since our own reindenting tool will remove them eventually anyway)